plenoptic/synthesize/eigendistortion.py

```diff
@@ -161,30 +165,24 @@ def to(self, *args, attrs=[], **kwargs):
         """Send attrs to specified device. See docstring of Synthesis.to()"""
         super().to(*args, attrs, **kwargs)
 
-    def synthesize(self, method='power', e_vecs=None, tol=1e-10, n_steps=1000, seed=0, debug_A=None,
-                   store_progress=False):
+    def synthesize(self, k=1, method='power', tol=1e-8, max_steps=1000, seed=0, store_progress=False):
         r"""Compute eigendistortions of Fisher Information Matrix with given input image.
 
         Parameters
         ----------
-        method: {'exact', 'power', 'lanczos'}, optional
+        method: {'exact', 'power', 'svd'}, optional
             Eigensolver method. Jacobian tries to do eigendecomposition directly (
             not recommended for very large matrices). 'power' (default) uses the power method to compute first and
-            last eigendistortions, with maximum number of iterations dictated by n_steps. 'lanczos' uses the Arnoldi
-            iteration algorithm to estimate the _entire_ eigenspectrum and thus more than just two eigendistortions,
-            as opposed to the power method. Note: 'lanczos' method is experimental and may be numerically unstable.
-            We recommend using the power method.
-        e_vecs: iterable, optional
-            Integer list of which eigenvectors to return for ``method='lanczos'``.
+            last eigendistortions, with maximum number of iterations dictated by n_steps. 'svd' uses randomized SVD
+            to approximate the top k eigendistortions and their corresponding eigenvalues.
+        k: int
+            How many vectors to return using block power method or svd.
```

Comment:
Parameters out of order in docstring. I think `method` should be first in the call signature, no? Since that's the most important one?

Reply:
done.